public/js/components/Graph.js

```diff
@@ -11,29 +11,31 @@ export default class Graph extends React.Component {
     }
 
     componentDidMount() {
-        // TODO: Use actuial data
-        fetch(`keywords/trump`)
-            .then(res => res.json())
-            .then((dataPoints) => {
-                dataPoints = dataPoints
-                    .sort((a, b) => a.times - b.times)
+        fetch(`keywords/top`)
+            .then(res => {
+                return res.json()
+            })
+            .then(words => {
+                // for now graphing first word associated with the top article
+                // createing array of dataPoints
```

Comment:
It's be obvious most of the time why things are needed, but these comments are talking about what whats happening